games-arcade/wormux/wormux-20130927.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A free Worms clone"
+DESCRIPTION="War has been declared! Battle your favorite free software mascots \  
+in the Wormux arena. With big sticks of dynamite, grenades, baseball bats, and \ 
+bazookas, you can exterminate your opponent in a 2D cartoon style scenery. \ 
+The goal of the game is to destroy all of your opponents' mascots."
+ 
+HOMEPAGE="https://github.com/a-team/wormux"
+COPYRIGHT="<a-team>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="e0dc503460055a14a0d7fd94e66d7484264ee62b"
+SOURCE_URI="https://github.com/a-team/wormux/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="wormux-$portVersion-$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d5e66808de2e4912a64b583e3be4b8ea52cd220b8617a113507cca2c2291c6d3"
+SOURCE_DIR="wormux-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wormux$secondaryArchSuffix = $portVersion
+"
```

Comment:
```diff
 PROVIDES="
 	wormux$secondaryArchSuffix = $portVersion
+	app:Wormux
-"
+	"
```
Remarks:
* I did not test the recipe, so I don't know if the runtime is called Wormux.
* The `Wormux` runtime should (probably) be installed in `$appsDir` otherwise the `cmd:Wormux` will not be valid.
* Regarding the name of the recipe, I just realized that `wormux-20130927.recipe` might not be a good choice after all. Something like `wormux-0~git20130927.recipe` would be safer, at least until upstream chooses a naming convention for the versions. (Having `20130927` as the version would make it impossible to have an automatic update to anything < 20130927.)
impossible